### PR TITLE
Fix/headerfix

### PR DESCRIPTION
**Pull request by Nkaka23dev**

### CODE REVIEW

**Review by arjorb — APPROVED** [review]
Looks good to me!

File: components/cards/BlogCard.jsx [hunk 1]
```diff
@@ -120,21 +120,21 @@ export default function BlogCard(props) {
...
-          <div class="flex-1 bg-white flex flex-col justify-between">
-            <div class="flex-1">
-              <p class="w-65.5 mt-1.25 font-Neufile text-desktopHeadingTwo font-normal leading-2.5 tracking-cardDescription text-primaryGrey md:w-102 lg:w-27.3125
+        <div className="flex flex-col  md:w-112 lg:w-29.8125 box-border px-5 w-75.5 rounded-xl border border-solid border-secondaryGrey shadow-2.5xl lg:px-35">
+          <div className="flex-1 bg-white flex flex-col justify-between">
+            <div className="flex-1">
+              <p className="w-65.5 mt-1.25 font-Neufile text-desktopHeadingTwo font-normal leading-2.5 tracking-cardDescription text-primaryGrey md:w-102 lg:w-27.3125
         ">
-                <a href="#" class="">02 Oct 2022 </a>
+                <a href="#" className="">02 Oct 2022 </a>
```
Comment (ishoborabyose) [inline components/cards/BlogCard.jsx:128]: THE CLASSNAME SHOULD NOT BE EMPTY

File: components/section/AboutSection/About.jsx [hunk 2]
```diff
@@ -15,8 +15,8 @@ export default function About (){
     const [activeTab, setActiveTab] = useState(0);
     return(
       <Wrapper>
-                <section className='hidden md:flex justify-end  py-24'>
-        <div className='hidden w-semi font-Neufile  md:block '>
+        <section className='hidden md:flex justify-end  py-24'>
+          <div className='hidden xs:w-21.875 md:w-[30.25rem] lg:w-[38rem] xl:w-47.3125 font-Neufile  md:block '>
    <div className=''>
```
Comment (ishoborabyose) [inline components/section/AboutSection/About.jsx:20]: Also here you should not left empty classname

File: components/cards/BlogCard.jsx [hunk 3]
```diff
@@ -5,21 +5,21 @@ export default function BlogCard(props) {
   return (
     <>
       <div className="flex gap-7 overflow-scroll pb-4 p-1.2813 py-8 md:pl-2.25   lg:pl-3.65 lg-sm:pb-10 cardScroll">
```
Comment (ishoborabyose) [inline components/cards/BlogCard.jsx:7]: You should not left the white space in your code, for best practice

**Review by ishoborabyose — APPROVED** [review]
## LGTM

**Review by haroldmud — APPROVED** [review]

**Pull request merged**